### PR TITLE
Add material-flux example

### PR DESCRIPTION
**Pull request by azu**
@voronianski HI! Thanks for creating nice project.

I've added an example implementation using [material-flux](https://github.com/azu/material-flux).
(it is similar to [Flummox](https://github.com/acdlite/flummox).)

gif

Please let me know if you need me to change anything! 
Thanks!

### CODE REVIEW

File: material-flux/js/Context.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
```
Comment (voronianski) [inline material-flux/js/Context.js]: @azu please remove `'use strict'` everywhere. It's unnecessary in ES6.

File: material-flux/js/actions/Actions.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+import { Action } from 'material-flux';
+import { checkoutProducts, getAllProducts } from '../utils/WebAPIUtils';
+export var keys = {
+    "getAllProducts": "getAllProducts",
```
Comment (voronianski) [inline material-flux/js/actions/Actions.js]: @azu please change to single quotes, see [CONTRIBUTING.md](https://github.com/voronianski/flux-comparison/blob/master/CONTRIBUTING.md).

File: material-flux/js/components/ProductsContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    },
+
+});
+
+let ProductsListContainer = React.createClass({
+    _getStateFromStores(){
+        let { context } = this.props;
+        return {
+            products: context.productsStore.getProducts()
+        }
```
Comment (voronianski) [inline material-flux/js/components/ProductsContainer.jsx]: @azu missed semicolon

**Comment by azu** [comment]
@voronianski Thanks for review!

I'v fixed the style issue and rebased the commit. 
- remove 'use strict'
- use single quotes
- add missed semicolon

**Comment by voronianski** [comment]
@azu merged

**Comment by azu** [comment]
@voronianski Thanks!

**Pull request merged**